hazelcast/src/test/java/com/hazelcast/internal/management/MCUpdateConfigOperationTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management;
+
+import com.hazelcast.client.impl.clientside.HazelcastClientProxy;
+import com.hazelcast.client.impl.protocol.codec.MCUpdateConfigCodec;
+import com.hazelcast.client.impl.spi.impl.ClientInvocation;
+import com.hazelcast.client.map.AbstractClientMapTest;
+import org.junit.Test;
+
+import java.util.concurrent.ExecutionException;
+
+public class MCUpdateConfigOperationTest
```

Comment:
please add a test for Reload operation too

Reply:
Well, since it isn't exactly a happy-path test (the operation gets successfully submitted then logs a `UnsupportedOperationException` to the console) it wasn't difficult to do the same for reload :)
But I'd spare the reload test in the EE repo.